frontend/app/services/SubscriptionService.scala

```diff
@@ -155,6 +156,17 @@ class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
     } yield subscriptionDetails
   }
 
+  def getUsageCountWithinTerm(memberId: MemberId, unitOfMeasure: String): Future[Int] = (for {
+    account <- getAccount(memberId)
+    subscriptions <- zuoraRestService.subscriptionsByAccount(account.id)
+    subscription = subscriptions.head if subscriptions.nonEmpty
+    startDate = formatDateTime(subscription.termStartDate)
+    whereClause = s"StartDateTime >= '$startDate' AND SubscriptionID = '${subscription.id}'"
```

Comment:
What Zuora service? We have both the REST and the SOAP one... This method uses both, so I have placed it in SubscriptionService